libzstd/encoder/Cargo.toml

```diff
@@ -7,4 +7,4 @@ edition = "2021"
 [lib]
 
 [dependencies]
-zstd = { git = "https://github.com/scroll-tech/zstd-rs", branch = "hack/mul-block", features = ["experimental"]}
\ No newline at end of file
+zstd = { version = "0.13", features = ["experimental"]}
```

Comment:
I'm concerned that this will affect the compression of codec version < 7, thus affecting the rollup verifier's batch hash calculation. @jonastheis, would the current L1 DA syncer replace constructing local blocks and compress them to a batch? e.g., replace by getting blob raw data and decompress it, then verify against local blocks.)

Reply:
Yes. I think the later is better designation and we should finally adopt to that. Making compression more than once and expect the result is identify seems a bit danger because the result would be affected by many factors: any update of minor version, configuration etc.

Reply:
Quite agree with the claim.